### PR TITLE
World Air Quality Index Handler

### LINKED ISSUE #8132: [Integration]: Integrate World Air Quality Index with MindsDB
### Is there an existing integration?

- [X] I have searched the existing integrations.

### Use Case

The goal of this issue is to build a new handler that will utilize the AQICN API to connect MindsDB to AQICN.

## Resources 💡
🏁 To get started, please refer to the following resources:

📚 [Building Application Handler docs](https://docs.mindsdb.com/contribute/app-handlers): This documentation will guide you through the process of building a custom handler for MindsDB. It contains essential information on the handler structure, methods, and best practices.

📖 [AQICN](https://aqicn.org/) - This is a free service.

👉 [Twitter Handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/twitter_handler): This existing handler connects MindsDB with the Twitter API, allowing users to automate tasks related to posting and analyzing tweets. You can use the Twitter Handler as a reference for building the Slack API integration.

👉 [Binance API Handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/binance_handler): Another example of a successful MindsDB integration to the Binance API to make trade data available to use for model training and predictions.

## Next Steps 🕵️‍♂️ 🕵️‍♀️
You can familiarize yourself with the [AQICN API](https://aqicn.org/json-api/doc/).
Create a new handler that connects MindsDB to AQICN API.
Implement the core features mentioned above.
Thoroughly test the integration to ensure proper functionality.
Submit a pull request with the new AQICN Integration handler.

## 📢 Additional rewards 🏆
Each App integration brings 5️⃣ 0️⃣ : points for SWAG and entry into the draw for a 👇 :

💻 [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
👕 🐻 [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)
ℹ️ For more info check out https://mindsdb.com/hacktoberfest/ 👈

### PR DESCRIPTION
**Pull request by Better-Boy**
## Description

World Air Quality Index Handler Implementation

Fixes #8132 

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

Screenshot 2023-10-26 at 9 01 57 PM
Screenshot 2023-10-26 at 9 27 35 PM
Screenshot 2023-10-26 at 9 35 54 PM

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Review by MinuraPunchihewa — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/aqicn_handler/aqicn_tables.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+            "data.time.v",
+            "data.time.iso",
+            "data.forecast.daily.o3",
+            "data.forecast.daily.pm10",
+            "data.forecast.daily.pm25",
+            "data.debug.sync"
+        ]
+
+
+class AQByNetworkStationTable(APITable):
```
[the remark this answers is not in the document]
Comment (Better-Boy) [reply mindsdb/integrations/handlers/aqicn_handler/aqicn_tables.py:336]: I forgot to register the table with the handler. I added the table and updated the readme.

File: mindsdb/integrations/handlers/aqicn_handler/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+CREATE DATABASE mindsdb_aqicn
+WITH ENGINE = 'aqicn',
+PARAMETERS = {
+  "api_key": "api_key"
+};
+~~~~
+
+Use the established connection to query your database:
+
+~~~~sql
```
[the remark this answers is not in the document]
Comment (Better-Boy) [reply mindsdb/integrations/handlers/aqicn_handler/README.md:62]: The query doesn't need any columns in where clause as it takes the IP address of the user querying the API.

[the remark this answers is not in the document]
Comment (MinuraPunchihewa) [reply mindsdb/integrations/handlers/aqicn_handler/README.md:62]: @Better-Boy I am not sure what you mean by this. What I was referring to is how, for example, the `city` column is required in the `WHERE` clause to run query the `AQByCityTable`.

**Review by MinuraPunchihewa — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/aqicn_handler/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Better-Boy) [reply mindsdb/integrations/handlers/aqicn_handler/README.md:62]: Ohk, the city column is used to get the air quality index for a given city. For more info - https://aqicn.org/json-api/doc/#api-City_Feed

[the remark this answers is not in the document]
Comment (MinuraPunchihewa) [reply mindsdb/integrations/handlers/aqicn_handler/README.md:62]: Yes, understood. I am just asking for such columns to be mentioned in the README along with the relevant examples.

[the remark this answers is not in the document]
Comment (Better-Boy) [reply mindsdb/integrations/handlers/aqicn_handler/README.md:64]: @MinuraPunchihewa Done, updated readme.

**Review by MinuraPunchihewa — APPROVED** [review]

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**